fapolicy_analyzer/ui/subject_list.py

```diff
@@ -141,6 +155,25 @@ def find_db_trust(subject):
         if changeset:
             dispatch(apply_changesets(changeset))
 
+    def __show_confirmation_dialog(self, subjects):
+        changeset = None
+        parent = self.get_ref().get_toplevel()
+        confirmationDialog = ConfirmChangeDialog(parent=parent).get_ref()
+        confirm = confirmationDialog.run()
+        confirmationDialog.destroy()
+
+        trust = self.find_db_trust(subjects[0])
```

Comment:
I think here you want to loop over all the subjects and add them to the `Changeset` right?